shogun-config/src/main/resources/application-base.yml

```diff
@@ -56,3 +56,23 @@ keycloakauth:
   password: shogun
   master-realm: master
   admin-client: admin-cli
+
+upload:
+  file:
+    supportedContentTypes:
```

Comment:
Can this be set to `all` or `*` in a childproject? Otherwise i would like to remove this restriction in shogun.

Reply:
Not yet, but I think this a very good idea. I'l check that.